site/content/docs/commands/secret-init.en.md

```diff
@@ -0,0 +1,96 @@
+
+## What's next?
+
+Copilot will create SSM parameters named `/copilot/<app name>/<env name>/secrets/<secret name>`. 
+Using the parameter names, you can then modify the `secrets` section in your [service's](https://aws.github.io/copilot-cli/docs/manifest/backend-service/#secrets) or [job's](https://aws.github.io/copilot-cli/docs/manifest/scheduled-job/#secrets) manifest to reference the secrets that are just created. 
+
+For example, suppose you have an application `my-app`, and you've created a secret `db_host` in `prod` and `dev` environments.
+You can modify your service's manifest as follows:
+```yaml
+environments:
+    prod:
+      secrets: 
+        DB_PASSWORD: /copilot/my-app/prod/secrets/db_password
+    dev:
+      secrets:
+        DB_PASSWORD: /copilot/my-app/dev/secrets/db_password
+```
+
+Once you deploy this update in manifest, your service or job will be able to access the environment variable `DB_PASSWORD`.
+It will have the value of the SSM parameter `/copilot/my-app/prod/secrets/db_password` if the service/job is deployed in a `prod` environment, and `/copilot/my-app/dev/secrets/db_password` if it's deployed in a `dev` environment.
+
+This works because ECS Agent will resolve the SSM parameter when it starts up your task, and set the environment variable for you.
+
+!!!info
+    Note that `/copilot/my-app/prod/secrets/db_password` is only accessible by any services or jobs that live in `my-app`'s `prod` environment, and similarly for `/copilot/my-app/dev/secrets/db_password`.
+
+## <span id="secret-init-cli-input-yaml">How do I use the `--cli-input-yaml` flag?</span>
+You can specify multiple secrets, and their values in each of your existing environments in a file. Then you can use the file as the input to `--cli-input-yaml` flag. Copilot will read from the file and create or update the secrets accordingly.
+
+The YAML file should be formatted as follows:
+```yaml
+<secret A>:
+  <env 1>: <the value of secret A in env 1>
+  <env 2>: <the value of secret A in env 2>
+  <env 3>: <the value of secret A in env 3>
+<secret B>:
+  <env 1>: <the value of secret B in env 1>
+  <env 2>: <the value of secret B in env 2>
+```
+
+Here is an example input file that creates secrets `db_host` and `db_password` in `dev`, `test` and `prod`, and `notification_email` in `dev` and `test` environments. Note that `notificaition-email` won't be created for `prod` environment since it doesn't have a value for `prod`.
+```yaml
+db_host:
+  dev: dev.db.host.com
+  test: test.db.host.com
+  prod: prod.db.host.com
+db_password:
+  dev: dev-db-pwd
+  test: test-db-pwd
+  prod: prod-db-pwd
+notification_email:
+  dev: dev@email.com
+  test: test@email.com
+```
+
+!!! attention 
+    Secrets are not supported for Request-Driven Web Services.
```

Comment:
Can we move this to the bottom of the "## What does it do section?" so that it's more apparent for folks